src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -179,6 +186,7 @@ public bool HasExited
         {
             get
             {
+                CheckDisposed();
```

Comment:
This and other similar places did not throw exception at all before this change, and will throw exception now. It is a potential breaking change.